pkg/controller/plan/kubevirt.go

```diff
@@ -1166,6 +1173,33 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 	return
 }
 
+// Attempt to find a suitable instancetype preference.
+func (r *KubeVirt) instancetype(vm *plan.VMStatus) (virtualMachine *cnv.VirtualMachine, ok bool) {
+	instancetypeName, err := r.Builder.InstanceTypeName(vm.Ref)
+	if err != nil {
+		return
+	}
+
+	instanceTypePreference := &instancetype.VirtualMachineClusterPreference{}
+	err = r.Destination.Client.Get(
+		context.TODO(),
+		client.ObjectKey{Name: instancetypeName},
+		instanceTypePreference)
+	if err != nil {
+		r.Log.Info("could not find instance type preference for destination VM.",
+			"vm",
+			vm.String(),
+			"error",
+			err)
```

Comment:
then it seems more appropriate to log it in error-level

Reply:
but we may have an actual error here..
I see your point in saying that it's not necessarily a severe error
I'd check the type of the error - if it's a not-found error, print an info message that no corresponding instance type was found, falling back... and if the error is different, log an error

Reply:
it changed to not found error - info, otherwise it will return a real error.